### PR TITLE
[AIG] How to add feedback documentatation

### PR DESCRIPTION
**Pull request by daisyfaithauma**
### Summary

- Initial documentation on how to add human feedback

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Review by kathayl — APPROVED** [review]

File: src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Human feedback is a valuable metric to assess the performance of your AI models. By incorporating human feedback, you can gain deeper insights into how the model's responses are perceived and how well it performs from a user-centric perspective. This feedback can then be used in evaluations to calculate performance metrics, driving optimization and ultimately enhancing the reliability, accuracy, and efficiency of your AI application.
+
+Human feedback measures the performance of your dataset based on direct human input. The metric is calculated as the percentage of positive feedback (thumbs up) given on logs, which are annotated in the Logs tab of the Cloudflare dashboard. This feedback helps refine model performance by considering real-world evaluations of its output.
+
+This tutorial will guide you through the process of adding human feedback to your evaluations in AI Gateway.
+
+## 1. Log into the dashboard
+
+1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account.
+2. Navigate to the AI Gateway section.
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx]:
```suggestion
2. Go to **AI** > **AI Gateway**.
```

File: src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    text: Beta
+---
+
+Human feedback is a valuable metric to assess the performance of your AI models. By incorporating human feedback, you can gain deeper insights into how the model's responses are perceived and how well it performs from a user-centric perspective. This feedback can then be used in evaluations to calculate performance metrics, driving optimization and ultimately enhancing the reliability, accuracy, and efficiency of your AI application.
+
+Human feedback measures the performance of your dataset based on direct human input. The metric is calculated as the percentage of positive feedback (thumbs up) given on logs, which are annotated in the Logs tab of the Cloudflare dashboard. This feedback helps refine model performance by considering real-world evaluations of its output.
+
+This tutorial will guide you through the process of adding human feedback to your evaluations in AI Gateway.
+
+## 1. Log into the dashboard
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx]:
```suggestion
## 1. Log in to the dashboard
```

File: src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account.
+2. Navigate to the AI Gateway section.
+
+## 2. Access the Logs tab
+
+1. In the dashboard, go to the Logs tab under the AI Gateway section.
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx]:
```suggestion
1. Go to **Logs**.
```

File: src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+   - Status: Whether the request was successful, cached, or failed.
+   - Model: The model used in the request.
+   - Tokens: The number of tokens consumed by the response.
+   - Cost: The cost based on token usage.
+   - Duration: The time taken to complete the response.
+   - Feedback: Where you can provide human feedback on each log.
+
+## 3. Provide human feedback
+
+1. In the Logs tab, click on the log entry you want to review. This expands the log, allowing you to see more detailed information.
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx]:
```suggestion
1. Click on the log entry you want to review. This expands the log, allowing you to see more detailed information.
```

File: src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+2. In the expanded log, you can view additional details such as:
+   - The user prompt.
+   - The model's response.
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx]:
```suggestion
   - The model response.
```

I literally just coincidentally read the style guide on [possessives](https://developers.cloudflare.com/style-guide/grammar/parts-of-speech/possessives/) lol. Sorry for being pedantic.

File: src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+   - HTTP response details.
+   - Endpoint information.
+3. You will see two icons:
+   - Thumbs up: Indicates positive feedback.
+   - Thumbs down: Indicates negative feedback.
+4. Click either the thumbs up or thumbs down icon based on how you rate the model's response for that particular log entry.
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx]:
```suggestion
4. Click either the thumbs up or thumbs down icon based on how you rate the model response for that particular log entry.
```

File: src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+## 4. Evaluate human feedback
+
+After providing feedback on your logs, it becomes part of the evaluation process.
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx]:
```suggestion
After providing feedback on your logs, it becomes a part of the evaluation process.
```

File: src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+:::note[Note]
+
+You need to select human feedback as an evaluator to receive its metrics.
+
+:::
+
+## 5. Review results
+
+After running the evaluation,review results on the Evaluations tab.
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx]:
```suggestion
After running the evaluation, review the results on the Evaluations tab.
```

File: src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+You will be able to see the model's performance based on cost, speed, and now human feedback, represented as the percentage of positive feedback (thumbs up).
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx]:
```suggestion
You will be able to see the performance of the model based on cost, speed, and now human feedback, represented as the percentage of positive feedback (thumbs up).
```

File: src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+The human feedback score is displayed as a percentage, showing how manyy of the dataset's responses were rated positively.
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx]:
```suggestion
The human feedback score is displayed as a percentage, showing the distribution of positively rated responses from the database.
```

I can't see an example of this in PCX dashboard account, so sorry if this is redundant, feel free to ignore.

Is it displayed as a percentage or a number? (Or both?) If it's percentage-only, then can a user still determine "how many" of the dataset's responses were rated positively?

**Review by Oxyjun — APPROVED** [review]

File: src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx (hunk 10 above)
Comment (daisyfaithauma) [reply src/content/docs/ai-gateway/evaluations/add-human-feedback.mdx]: Yeah you are correct

**Pull request merged**